easybuild/easyconfigs/l/likwid/likwid-4.2.0-intel-2017a.eb

```diff
@@ -0,0 +1,52 @@
+##
+# -*- mode: python; -*-
+# EasyBuild reciPY for LikWid -- see https://github.com/RRZE-HPC/likwid
+# as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2017 UL HPC -- hpc.uni.lu
+# Authors::   UL HPC Team <hpc-sysadmins@uni.lu>
+# License::   MIT/GPL
+#
+#
+easyblock = 'ConfigureMake'
+
+name = 'likwid'
+version = '4.2.0'
+
+homepage = 'https://github.com/RRZE-HPC/likwid'
+description = """Likwid stands for 'Like I knew what I am doing'. Likwid is a
+simple to install and use toolsuite of command line applications for performance
+oriented programmers. It works for Intel and AMD processors on the Linux
+operating system."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/RRZE-HPC/likwid/archive/']
+sources = ['%(version)s.tar.gz']
+
+patches = ['likwid-%(version)s-config-mk.patch']
+
+builddependencies = [('binutils', '2.27')]
```

Comment:
this is not required when using the `intel` toolchain, since it will be loaded already